backend/service-lib/src/main/kotlin/calls/ApiConventions.kt

```diff
@@ -602,7 +602,7 @@ verification API to cleanup these resources later.
 @UCloudApiStable
 data class BulkRequest<out T : Any>(val items: List<T>) {
     init {
-        if (items.size > 1_000 || items.isEmpty()) throw RPCException("BulkRequest must not be empty", HttpStatusCode.BadRequest)
+        if (items.size > 1_000 || items.isEmpty()) throw RPCException("BulkRequest must not be empty or exceed limit", HttpStatusCode.BadRequest)
```

Comment:
This will be shown as an error message on the frontend, right? This is not a great error message to show the user. With or with the "or exceed limit" part.

Reply:
Hopefully, this would only occur when doing development, but if it gets through, the user won't know what to do different.